package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "nodemon server/server.js",
     "start": "node server/server.js",
-    "test": "mocha --exit --timeout 10000",
+    "test": "mocha --exit --timeout 10000 test/mocha/*.js",
```

Comment:
Add `test/mocha/*.js` in line no 16
It should be 
    `"test-coverage": "nyc mocha --exit --timeout 10000 test/mocha/*.js"`

Reply:
Okay Thanks @jeffin143